schema.graphql

```diff
@@ -436,6 +436,30 @@ type Auction @entity {
   deleteAt: BigInt!
 }
 
+type DebtAuction implements Auction @entity {
+  id: ID!
+  bidAmount: BigInt!
+  quantity: BigInt!
+  highestBidder: Bytes!
+  endTime: BigInt!
+  createdAt: BigInt!
+  lastUpdate: BigInt!
+  active: Boolean!
+  deleteAt: BigInt!
+}
+
+type SurplusAuction implements Auction @entity {
+  id: ID!
+  bidAmount: BigInt!
+  quantity: BigInt!
+  highestBidder: Bytes!
+  endTime: BigInt!
+  createdAt: BigInt!
+  lastUpdate: BigInt!
+  active: Boolean!
+  deleteAt: BigInt!
+}
+
 " Maker Vault (formerly collateralized debt position or CDPs) "
```

Comment:
it will be better to have the comments here too. So it would be shown in the graph playground.

Reply:
I guess playground will inherit comments from the interface